src/main/java/com/example/HR_AppJava/HR_App/Main.java

```diff
@@ -0,0 +1,30 @@
+package com.example.HR_AppJava.HR_App;
+
+import java.util.Scanner;
+
+public class Main {
+    Scanner scanner = new Scanner(System.in);    
+    static Person p1 = new Person();
```

Comment:
If you create the class objects inside the main method, you don't have to be static.